### DISCUSSION TITLE
posts/027-buy-taobao-to-jp/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#14] [Announcements]
# posts/027-buy-taobao-to-jp/

就算不逃稅，同樣的東西在淘寶上買就是比你日便宜太多😁👍🏻

**Comment by PermeationLoop** [comment]
最近又想海淘 Mac Mini 的 SSD 擴容配件了😁👍🏻

**Comment by PermeationLoop** [comment]
一個星期前俺海淘了一臺 Custom build 的 Mac mini，也就是更改了默認配置，需要在工廠製造那種，直到目前都 tm 沒有發貨．
俺還海淘了一個外接 SSD 硬盤盒子，是從中國發貨的，兩三天前的追蹤就已經到達中國的機場了，這次不會™️的又要運 19 天把🥶👎🏻

**Comment by PermeationLoop** [comment]
我很惱啊，我要草飼牠們的馬

**Comment by ChiuHuang** [comment]
笑死
